docs/uninstall.md

```diff
@@ -0,0 +1,449 @@
+autoscaling decisions.
+The following command will install the components needed to support HPA-class autoscaling:
+
+```bash
+kubectl delete -f {{< artifact repo="serving" file="serving-hpa.yaml" >}}
+```
+
+<!-- TODO(https://github.com/knative/docs/issues/2152): Link to a more in-depth guide on HPA-class autoscaling -->
+
+{{< /tab >}}
+
+{{% tab name="TLS with cert-manager" %}}
+
+1. Uninstall the component that integrates Knative with cert-manager:
+
+   ```bash
+   kubectl delete -f {{< artifact repo="net-certmanager" file="release.yaml" >}}
+   ```
+
+1. Optional: if you no longer need cert-manager, uninstall it by following the steps in the
+[cert-manager documentation](https://cert-manager.io/docs/installation/uninstall/kubernetes/).
+
+   {{< /tab >}}
+
+{{% tab name="TLS via HTTP01" %}}
+
+Uninstall the `net-http01` controller by running:
+
+```bash
+kubectl delete -f {{< artifact repo="net-http01" file="release.yaml" >}}
+```
+
+{{< /tab >}}
+
+{{% tab name="TLS wildcard support" %}}
+
+Uninstall the components needed to provision wildcard certificates in each namespace by running:
+
+```bash
+kubectl delete -f {{< artifact repo="serving" file="serving-nscert.yaml" >}}
+```
+
+{{< /tab >}}
+
+{{% tab name="DomainMapping CRD" %}}
+
+To uninstall the `DomainMapping` CRD run:
+
+```bash
+kubectl delete -f {{< artifact repo="serving" file="serving-domainmapping.yaml" >}}
+kubectl delete -f {{< artifact repo="serving" file="serving-domainmapping-crds.yaml" >}}
+```
+
+{{< /tab >}} {{< /tabs >}}
+
+
+### Uninstalling Magic DNS
+
+If you installed Magic DNS (xip.io), uninstall it by running:
+
+```
+kubectl delete -f https://storage.googleapis.com/knative-nightly/serving/latest/serving-default-domain.yaml
+```
+
+### Uninstalling a networking layer
+
+Follow the relevant procedure to uninstall the networking layer you installed:
+
+<!-- TODO: Link to document/diagram describing what is a networking layer.  -->
+<!-- This indentation is important for things to render properly. -->
+
+   {{< tabs name="serving_networking" default="Kourier" >}}
+   {{% tab name="Ambassador" %}}
+
+The following commands install Ambassador and enable its Knative integration.
+
+1. Uninstall Ambassador by running:
+
+   ```bash
+   kubectl delete --namespace ambassador \
+    -f https://getambassador.io/yaml/ambassador/ambassador-crds.yaml \
+    -f https://getambassador.io/yaml/ambassador/ambassador-rbac.yaml \
+    -f https://getambassador.io/yaml/ambassador/ambassador-service.yaml
+   ```
+
+1. Delete the Ambassador namespace:
+
+   ```bash
+   kubectl delete namespace ambassador
+   ```
+
+{{< /tab >}}
+
+{{% tab name="Contour" %}}
+
+The following commands install Contour and enable its Knative integration.
+
+1. Uninstall the Knative Contour controller by running:
+
+    ```bash
+    kubectl delete -f {{< artifact repo="net-contour" file="net-contour.yaml" >}}
+    ```
+
+1. Install a properly configured Contour:
```

Comment:
s/Install/Uninstall/

Probably don't need to say "properly configured" here.

Reply:
Thanks, will fix.